lucille-plugins/lucille-tika/src/main/java/com/kmwllc/lucille/tika/stage/TextExtractor.java

```diff
@@ -191,7 +191,7 @@ public void parseInputStream(Document doc, InputStream inputStream) {
           || (metadataWhitelist != null && metadataWhitelist.contains(cleanName))
           || (metadataWhitelist == null) && (metadataBlacklist == null)) {
         for (String value : metadata.getValues(name)) {
-          doc.addToField(newMetadataPrefix + cleanName, value);
+          doc.setOrAdd(newMetadataPrefix + cleanName, value);
```

Comment:
The PR was just to change the code at line 185 from addToField to setOrAdd, but I think the same logic should be applied here on line 194 as well -- field should only be multivalued if it contains two or more values